pages/news/2020-05-05-mixxx-gsoc-projects-2020.html

```diff
@@ -0,0 +1,36 @@
+title: Announcing our Google Summer of Code Projects 2020
+author: Holzhaus
+date: 2020-05-05 01:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+Mixxx is participating in this year's [Google Summer of Code (GSoC)](https://summerofcode.withgoogle.com/), an opportunity for students to work on Mixxx full-time as a 3-month summer project and receive a stipend from Google.
+
+For this year's Google Summer of Code, Mixx has been [granted 3 slots](https://summerofcode.withgoogle.com/organizations/6577142898360320/) that will be filled by students from 3 different continents:
+[Harshit Maurya](https://github.com/hacksdump) from India, [Cristiano Lacerda](https://github.com/crisclacerda) from Brazil and [Janek Fischer](https://github.com/xerus2000) from Germany.
+Congratulations!
+
+Harshit and Cristiano will be working on making a new beatgrid format that marks bars, meter, and phrases.
+
+Cristiano will focus on [implementing autodetection](https://summerofcode.withgoogle.com/projects/#5309419128094720) of this information from audio files.
+This DSP- and research-heavy project will mentored by [Daniel Schürman](https://github.com/daschuer).
+
+Harshit will [focus on creating the user interface](https://summerofcode.withgoogle.com/projects/#5081784049467392) for viewing and editing the beatgrid and using this information in other Mixxx features like waveforms, loops, beatjumps and quantization.
+[Uwe Klotz](https://github.com/uklotzde) will be his primary mentor.
+
+Janek will be working on a [macro recording feature](https://summerofcode.withgoogle.com/projects/#4641898733502464) inspired by [Serato Flip](https://serato.com/dj/pro/expansions/flip) to record custom edits in Mixxx without altering the audio files.
+His project will be mentored by [Jan Holthuis](https://github.com/Holzhaus).
+
+We're going to do things a bit differently for GSoC this year than we have in the past.
+Before, mentors have had a lot of private one-on-one communication with students.
+This year we'll be encouraging students to discuss their work more publicly so the whole community is involved throughout their projects.
+
+Thus, if you're interested in these projects check our [development](https://mixxx.zulipchat.com/) for discussions and progress reports.
```

Comment:
```suggestion
Thus, if you're interested in these projects check our [Zulip chat](https://mixxx.zulipchat.com/) for discussions and progress reports.
```

Reply:
development chat?

Reply:
Or "developers chat"? But the chat is not only for developers, also for users.

Reply:
Just "chat" then?

Reply:
Why not "Zulip chat"?

Reply:
Ok